bin/si/src/ai-agent/mcp-server/tools/componentDiscover.ts

```diff
@@ -200,6 +207,15 @@ export function componentDiscoverTool(server: McpServer) {
               },
             });
 
+            // if discovery failed, remove the component
+            if (discoverResponse.status === 500) {
```

Comment:
Good catch. Why not `discoverResponse.status !== 200`? Is it because certain non-200 codes are fine?